library/src/main/java/com/wultra/android/mtokensdk/push/PushService.kt

```diff
@@ -86,24 +95,14 @@ class PushService(powerAuthSDK: PowerAuthSDK, appContext: Context, okHttpClient:
         )
     }
 
-    /**
-     * Registers HMS on backend to receive notifications about operations
-     * @param hmsToken Huawei Push Token
-     * @param callback Result listener
-     */
-    fun registerHuawei(hmsToken: String, callback: (result: Result<Unit>) -> Unit) {
-        pushApi.registerToken(
-            PushRegistrationRequest(PushRegistrationRequestObject(hmsToken, Platform.HUAWEI)),
-            object : IApiCallResponseListener<StatusResponse> {
-                override fun onSuccess(result: StatusResponse) {
-                    callback(Result.success(Unit))
-                }
+    // deprecated API
+    @Deprecated("Use register function with `data` parameter as a replacement")
```

Comment:
Use `@Deprecated("This method is deprecated since the server version 1.10.0. Use `register` with `data` parameter as a replacement") // deprecated in 1.13.0`